{{ cookiecutter.repo_name }}/DESCRIPTION

```diff
@@ -0,0 +1,22 @@
+Package: govcookiecutter
+Title: A Template Data Science Project
+Version: 0.0.0.9000
+Authors@R:
+    person(given = "First",
+           family = "Last",
+           role = c("aut", "cre"),
+           email = "first.last@example.com")
+Description: What the package does (one paragraph).
+License: `use_mit_license()`, `use_gpl3_license()` or friends to
+    pick a license
```

Comment:
Would strongly recommend using Jinja2 templating to automatically fill these values that are supplied by `cookiecutter.json` or pre-defined.  Not sure how to define an organisation, so followed this [SO answer](https://stackoverflow.com/a/50688358).

```suggestion
Package: {{ cookiecutter.repo_name }}
Title: {{ cookiecutter.project_name }}
Version: {{ cookiecutter.project_version }}
Authors@R:
    person(given = "{{ cookiecutter.organisation_name }}",
           role = c("aut", "cre"),
           email = "{{ cookiecutter.contact_email }}")
Description: {{ cookiecutter.overview }}
License: Unless stated otherwise, the codebase is released under 
    the MIT License. This covers both the codebase and any sample 
    code in the documentation. The documentation is © Crown 
    copyright and available under the terms of the Open Government 
    3.0 licence.
```